core/util.js

```diff
@@ -329,7 +329,7 @@ Util.Features = {xpath: !!(document.evaluate), air: !!(window.runtime), query: !
         'presto': detectPresto(),
         'trident': detectTrident(),
         'webkit': detectInitialWebkit(),
-        'gecko': detectGecko(),
+        'gecko': detectGecko()
```

Comment:
this is more a matter of personal taste, but the trailing comma makes diffs that add addition options (as well as blame logs) better, since adding a new line only shows the new line being added, instead of also showing the old line having a comma added as well.